runtime/runtime/tests/runtime_group_tools/mod.rs

```diff
@@ -93,16 +93,18 @@ impl StandaloneRuntime {
             &genesis,
             account_ids,
         );
-        let congestion_info = if ProtocolFeature::CongestionControl.enabled(PROTOCOL_VERSION) {
-            genesis
-                .config
-                .shard_layout
-                .shard_ids()
-                .map(|shard_id| (shard_id, ExtendedCongestionInfo::default()))
-                .collect()
-        } else {
-            Default::default()
-        };
+        #[allow(deprecated)]
+        let congestion_info =
+            if ProtocolFeature::_DeprecatedCongestionControl.enabled(PROTOCOL_VERSION) {
```

Comment:
Assume `ProtocolFeature::_DeprecatedCongestionControl.enabled(PROTOCOL_VERSION)` is always true and simplify block.